src/components/TableOfContents/styles.tsx

```diff
@@ -133,6 +133,7 @@ export const H3AnchorStyle = styled.div`
 export const HeaderStyle = styled.div`
   padding-right: 1.5rem;
   padding-bottom: 0.5rem;
+  padding-left: 1.5rem;
   border-left: 0.05rem solid var(--border-color);
 
   ${MQDesktop} {
```

Comment:
If this is going to be the desired behavior at all screen sizes can we remove this media query that is adding the same rule in for desktop

Reply:
Thank you, updated